fortran95.html.markdown

```diff
@@ -0,0 +1,445 @@
+---
+language: Fortran
+contributors:
+    - ["Robert Steed", "https://github.com/robochat"]
+filename: learnfortran.f95
+---
+
+Fortran is one of the oldest computer languages. It was developed in the 1950s 
+by IBM for numeric calculations (Fortran is an abreviation of "Formula 
+Translation"). Despite its age, it is still used for high-performance computing
+such as weather prediction. However, the language has changed considerably over
+the years, although mostly maintaining backwards compatibility; well known
+versions are FORTRAN 77, Fortran 90, Fortran 95, Fortran 2003, Fortran 2008 and
+Fortran2015.
+
+This overview will discuss the features of Fortran 95 since it is the most
+widely implemented of the more recent specifications and the later versions are
+largely similar (by comparison FORTRAN 77 is a very different language).
+
+```fortran
+
+! This is a comment.
+
+! Code can only exist inside programs, functions, subroutines or modules.
+
+! long lines can be continued using the continuation character '&'
+
+program example   !declare a program called example.
+
+    ! Indentation is not actually required but is recommended.
+
+    ! Declaring variables
+    ! ===================
+    
+    ! All declarations must come before statements and expressions.
+    
+    implicit none    !prevents dynamic declaration of variables (recommended!)
+    ! Implicit none must be redeclared in every function/program/module...
+    
+    ! IMPORTANT - Fortran is case insensitive.
+    real z
+    REAL Z2
+
+    real :: v,x    ! WARNING: default initial values are compiler dependent!    
+    real :: a = 3, b=2E12, c = 0.01
+    integer :: i,j,k=1,m
```

Comment:
Do you mean on line 45? Does this really matter? In real code consistency is useful but in a tutorial showing variety demonstrates what the compiler will accept.